lib/graphql/schema/unique_within_type.rb

```diff
@@ -1,14 +1,21 @@
 module GraphQL
   class Schema
     module UniqueWithinType
-      DEFAULT_SEPARATOR = "-"
+      class << self
+        attr_accessor :default_id_separator
+      end
+      self.default_id_separator = "-"
 
       module_function
 
+      def set_default_separator(separator)
+        self.default_id_separator = separator
+      end
```

Comment:
With an `attr_accessor`, there's not really a benefit to the extra setter method, is there? `GraphQL::Schema::UniqueWithinType.default_id_separator=` would be a public interface